ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeRaoParameters.java

```diff
@@ -49,6 +53,9 @@
     private PreventiveRaoStopCriterion preventiveRaoStopCriterion = DEFAULT_PREVENTIVE_RAO_STOP_CRITERION;
     private CurativeRaoStopCriterion curativeRaoStopCriterion = DEFAULT_CURATIVE_RAO_STOP_CRITERION;
     private double curativeRaoMinObjImprovement = DEFAULT_CURATIVE_RAO_MIN_OBJ_IMPROVEMENT; // used for CurativeRaoStopCriterion.PREVENTIVE_OBJECTIVE and CurativeRaoStopCriterion.PREVENTIVE_OBJECTIVE_AND_SECURE
+    private Map<String, Integer> maxCurativeTopoPerTso = new HashMap<>();
+    private Map<String, Integer> maxCurativePstPerTso = new HashMap<>();
+    private Map<String, Integer> maxCurativeRaPerTso = new HashMap<>();
```

Comment:
could you update SearchTreeRaoParameterConfigLoader for these new parameters

Reply:
yes I forgot it